aztec/src/main/kotlin/org/wordpress/aztec/source/Format.kt

```diff
@@ -324,20 +325,36 @@ internal object Format {
                 val spanEnd = text.getSpanEnd(it)
 
                 if (text[spanStart] == '\n' && text.getSpans(spanEnd, spanEnd + 1, IAztecParagraphStyle::class.java)
-                        .filter { it !is ParagraphSpan && text.getSpanStart(it) == spanEnd }.isEmpty()) {
+                        .filter { (it !is ParagraphSpan || !it.attributes.isEmpty()) && text.getSpanStart(it) == spanEnd }.isEmpty()) {
```

Comment:
Just to double check, this only retains the `<p>` if it contains at least one attribute, right?

Reply:
That's right.

Reply:
Ok, I updated `testRetainParagraphFormatting` in 90b4b7e